src/common/cockpitpipetransport.c

```diff
@@ -155,14 +155,10 @@ on_pipe_close (CockpitPipe *pipe,
           status = cockpit_pipe_exit_status (pipe);
           if (WIFSIGNALED (status) && WTERMSIG (status) == SIGTERM)
             problem = "terminated";
-          else if (WIFEXITED (status) && WEXITSTATUS (status) == 5)
```

Comment:
Those were old sshpass error codes, from before we used libssh. I'll split this change into another commit.